nampower/main.cpp

```diff
@@ -150,8 +152,8 @@ bool CastSpellHook(hadesmem::PatchDetourBase *detour, void *unit, int spellId, v
     }
 
     auto const cursorMode = *reinterpret_cast<int *>(Offsets::CursorMode);
-
-    if (ret && !!spell && !(spell->Attributes & game::SPELL_ATTR_RANGED) && cursorMode != 2)
+                                                                        //JT: cursorMode == 2 is for clickcasting
+    if (ret && !!spell && !(spell->Attributes & game::SPELL_ATTR_RANGED)/* && cursorMode != 2*/)
         BeginCast(currentTime, castTime, spellId);
```

Comment:
What does removing this condition give us?

Reply:
Enables nampower with clickcasting. When healing you can use an addon such as clique or LunaUnitFrames and configure the healing to go off on say clicking the right mouse button. W/o this change - nampower only worked with keybinds, not with clickcasting.